conjure/src/test/java/com/palantir/conjure/cli/ConjureCliTest.java

```diff
@@ -46,10 +47,15 @@ public void before() throws IOException {
 
     @Test
     public void correctlyParseArguments() {
-        String[] args = {"compile", inputFile.getAbsolutePath(), outputFile.getAbsolutePath()};
+        String[] args = {
+                "compile",
+                inputFile.getAbsolutePath(),
+                outputFile.getAbsolutePath(),
+                "--extensions", "{\"foo\": \"bar\"}"};
```

Comment:
thoughts on passing the extensions as a file instead? it might make it easier to debug if you could open up a directory locally and look at the json?

Reply:
I actually thought about that but preferred the inline JSON since it makes it more usable as a CLI. It also simplifies the gradle conjure implementation. From a debugability perspective, we log all of the arguments being passed to the compile in gradle and so it should be easy understand what was sent over

Reply:
ok fair enough